prisma/migrations/02_views/migration.sql

```diff
@@ -0,0 +1,7 @@
+
+CREATE ALGORITHM=UNDEFINED DEFINER=`root`@`%` SQL SECURITY DEFINER VIEW `Recurring_no_kid_bank_donors` AS select `Donors`.`ID` AS `DonorID`,`Donations`.`KID_fordeling` AS `KID`,count(0) AS `NumDonations`,`Donors`.`full_name` AS `DonorName`,(select `SubDonations`.`sum_confirmed` from `Donations` `SubDonations` where ((`SubDonations`.`KID_fordeling` = `Donations`.`KID_fordeling`) and (`SubDonations`.`Payment_ID` = 5)) order by `SubDonations`.`timestamp_confirmed` desc limit 1) AS `LatestSum` from (`Donations` join `Donors` on((`Donations`.`Donor_ID` = `Donors`.`ID`))) where (`Donations`.`Payment_ID` = 5) group by concat(`Donations`.`Donor_ID`,'-',`Donations`.`KID_fordeling`),`Donors`.`full_name`,`Donors`.`ID`,`Donations`.`KID_fordeling` having (`NumDonations` > 2) order by `NumDonations` desc;
```

Comment:
This content looks to be disabled when dumping the current schema :thinking: I wonder if it's needed. If it is, we should run a formatter on it so we can actually read it :grin:

Reply:
Formatted the sql files!